src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -76,25 +78,21 @@ public AutonModes(DriveBaseSubsystem d, IntakeSubsystem i, CDSSubsystem c) {
     initializeCommandGroups();
   }
 
+  // all subsystems come into play
   public AutonModes(
       DriveBaseSubsystem d,
       ShooterSubsystem s,
       LimelightSubsystem l,
       CDSSubsystem c,
       IntakeSubsystem i) {
 
-    System.out.println("---------Going through right constructor.");
-
     this.driveBaseSubsystem = d;
     this.shooterSubsystem = s;
     this.limelightSubsystem = l;
     this.cdsSubsystem = c;
     this.intakeSubsystem = i;
 
-    allSubsystemsEnabled = true;
-
-    // parameters are route names (names based on pathweaver, they are located under Auton file)
-    initializeTrajectories();
+    shooterEnabled = true;
```

Comment:
If this is false, we'll have a handful of auton modes with no ramsete commands that are still choosable. We probably don't need it for competition at all, but at minimum we should control this in a more visible way than a private variable in this class. Publish to dashboard, tie to modular subsystem, etc.